.pre-commit-config.yaml

```
@@ -1,13 +1,12 @@
 ---
 repos:
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.19.0
+  rev: v1.20.0
```

```suggestion
  rev: v1.24.0
```
This version uses the new `terraform-docs`. You'll need to run `brew update terraform-docs` for it to work